### ISSUE TITLE
Google Drive PDFs loading forever in brave-beta

### ISSUE THREAD

**Issue by Rohit-Rangaraj** [brave/brave-browser#19938]
## Description 
When I try to open pdfs from google drive, it just keeps loading it up forever.

## Steps to Reproduce

   1. Acquire some PDF file hosted on Google Drive
   2. Open it in brave-beta

## Actual result:
Just keeps loading forever.

## Expected result:
PDF file to load normally

## Reproduces how often: 
Every time I open up a PDF from Google Drive

## Brave version (brave://version info)
Brave | 1.33.93 Chromium: 96.0.4664.45 (Official Build) beta (64-bit)
Revision | 76e4c1bb2ab4671b8beba3444e61c0f17584b2fc-refs/branch-heads/4664@{#947}
OS | Linux

## Version/Channel Information:
brave-beta-bin from the Arch User Repository

- Can you reproduce this issue with the current release? 
- No the current release works perfectly well

- Can you reproduce this issue with the beta channel? 
- Yes

- Can you reproduce this issue with the nightly channel? 
- Haven't tried it on nightly

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? 
- No
- Does the issue resolve itself when disabling Brave Rewards? 
- No
- Is the issue reproducible on the latest version of Chrome? 
- No

## Miscellaneous Information:
Operating System: Arch Linux
Package used: brave-beta-bin from Arch User Repository
Link to package: https://aur.archlinux.org/packages/brave-beta-bin/

**Comment by rebron** [comment]
@Rohit-Rangaraj Can you try with shields down? Also, is it specific to the .pdf files? Are they larger than usual?

**Comment by Rohit-Rangaraj** [comment]
@rebron I tried it with shields down, still the same.
It isn't specific to .pdf files, but rather any file preview isn't working.
They are of the usual size. Just a few megabytes.

**Comment by rebron** [comment]
@Rohit-Rangaraj Are you only seeing this slow down of .pdf viewing from Google Drive? And the .pdf does load eventually correct, just longer than expected?

**Comment by rebron** [comment]
Wanted to check to see if there was a difference whether or not you're logged into Google.

**Comment by Rohit-Rangaraj** [comment]
> @Rohit-Rangaraj Are you only seeing this slow down of .pdf viewing from Google Drive? And the .pdf does load eventually correct, just longer than expected?
> 

Yes, only Google drive files are loading forever. The file doesn't load at all. I've let it alone for about 5 hours and still nothing. The issue happens for any file format opened in Google Drive except for Docs, Sheets and Slides.

**Comment by Rohit-Rangaraj** [comment]
> Wanted to check to see if there was a difference whether or not you're logged into Google. 

I've tried it both ways, and also on incognito and in Tor. Same thing happens.

**Comment by Rohit-Rangaraj** [comment]
@rebron Any updates?

**Comment by rebron** [comment]
@Rohit-Rangaraj This isn't a quick fix and still need to investigate the cause. It's Google Drive specific. Does switching this pref help: brave://settings/content/pdfDocuments to downloads instead of view in Brave.

**Comment by Rohit-Rangaraj** [comment]
@rebron, I understand. I tried changing it to download pdfs, but that doesn't seem to change anything because the files are previews built into the Google Drive website and not separate pdfs that are problematic.

**Comment by Rohit-Rangaraj** [comment]
@rebron When can I expect progress on this? Just checking in, I understand it's not easy to fix this.
